controllers/fleetnotification/fleetnotification_controller.go

```diff
@@ -37,10 +44,6 @@ var log = logf.Log.WithName("controller_fleetnotification")
 
 var _ reconcile.Reconciler = &ManagedFleetNotificationReconciler{}
 
-//+kubebuilder:rbac:groups=ocmagent.managed.openshift.io,resources=managedfleetnotifications,verbs=get;list;watch;create;update;patch;delete
-//+kubebuilder:rbac:groups=ocmagent.managed.openshift.io,resources=managedfleetnotifications/status,verbs=get;update;patch
-//+kubebuilder:rbac:groups=ocmagent.managed.openshift.io,resources=managedfleetnotifications/finalizers,verbs=update
```

Comment:
We should maybe leave this in, in the event of future possibility where we try and get kustomize producing the operator resources again. Unless there is some specific reason to get rid of it?